.github/workflows/verify-program.yml

```
@@ -0,0 +1,103 @@
+    name: Program / Verify
+    runs-on: ubuntu-latest-16-cores
+    permissions:
+      contents: read
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.git_ref }}
+
+      - name: Load environment variables
+        run: cat .github/.env >> $GITHUB_ENV
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:21: Double quote to prevent globbing and word splitting

(shellcheck)